EmployeeDirectory.DAL/DataOperations/DeptProjectOperations.cs

```diff
@@ -0,0 +1,36 @@
+﻿using EmployeeDirectory.DAL.Contracts.Providers;
+using Microsoft.Data.SqlClient;
+
+namespace EmployeeDirectory.DAL.DataOperations
+{
+    public class DeptProjectOperations(IDBConnection dBConnection):IDeptProjectOperations
```

Comment:
Use suffix as "Repository" in class names of DAL providers.
For example:  EmployeeRepository